application.md

```diff
@@ -1,5 +1,141 @@
+meetings with their mentors as well as weekly blog posts on the OpenAstronomy
+site describing their progress.
+They will also be invited to participate in the creation of posters,
+presentations and/or tutorials to show in upcoming scientific conferences.
+Students (as all contributors) will be required to write up a user guide and
+full documentation for their contribution.
+This will be reviewed by the community through the GitHub and BitBucket comment
+and annotation pull request platform.
+
+If the work done leads to a research publication, the student will be invited to
+collaborate in the preparation of the publication.
+These guidelines have already been successful applied as part of our previous
+participations in GSoC and the ESA Summer of Code programmes.
+
+[1] https://openastronomy.github.io/gsoc2015/student_guidelines.html
+
+
+## How will you keep students involved with your community after GSoC?
+
+We aim to involve all new people with our communities as much as we can.
+GSoC students will be encouraged no less.
+We hope to convince them of the attractive and usability that our projects have
+for the long term.
+As many solar and astrophysicists are new to Python, we will invite the students
+to give tutorials about their contribution to astronomy and astrophysics
+institutes near their home location.
+This will help the students get a better understanding of the needs in the
+community and to get closer to the users and the field.
+This may provide the student a big push in their career, and open new doors in
+the astronomical research world.
+Over the years various students have been giving talks at astronomical
+institutes in Austria, England, Ireland and India.
+
+To further encourage students, we will follow the example set by other GSoC
+projects (such as SymPy) and ask the students to spend a small fraction of their
+time reviewing others’ pull requests.
+We will ask them to help review one pull request per week, which should be no
+more than a 5% time commitment.
+
+## Has your org been accepted as a mentoring org in Google Summer of Code before?
+
+No
+
+## Are you part of a foundation/umbrella organization?
+
+No
+
+## What year was your project started?
+
+- [x] 2015
+
+## Anything else
+
+OpenAstronomy started in 2015 as an umbrella organization for GSoC and to coordinate other activities as workshops etc...
+The organizations under the umbrella has been created at different times, SunPy 2010, Astropy 2011, Julia Astro 2014, chiantipy 201?, yt 20XX, ...
+SunPy, Astropy and yt has participated previously on GSOC under the PSF and NumFocus umbrella. 
+From SunPy David and Stuart has been organizing since 2013, for Astropy ... has done so and for Yt Matthew has ...
+
+
+
+
 ## If you did not choose "veteran" in the checkbox, have you applied in the past? If so, for what year(s)?
 
 This organisation is new and this is our first year applying as such, however,
```

Comment:
Oh, maybe *this* is where you say the thing about us having done it in umbrella orgs.